Don't install changelogs for they're too heavy and are a git logs (instead of real changelogs) anyway
--- Makefile.am.orig	2019-07-26 18:19:11 UTC
+++ Makefile.am
@@ -12,7 +12,7 @@ readmedir = @readmedir@
 if HAVE_MARKDOWN
 readme_DATA = README.md COPYING ChangeLog TODO README.lirc AUTHORS README.html ChangeLog.html
 else
-readme_DATA = README.md COPYING ChangeLog TODO README.lirc AUTHORS ChangeLog.html
+readme_DATA = README.md TODO README.lirc AUTHORS
 endif
 
 desktopdir = $(datadir)/applications
